### ISSUE TITLE
Feature Request - cyclic list scrolling

### ISSUE THREAD

**Issue by svenpetersen1965** [pi1541/Pi1541#126]
It is probably pretty simple, but would be helpful in longer lists:
when you are on top of the list and press the up key, it would be nice to go to the end of the list. When you are at the end of the list and you press the down key, it should go to the top of the list.

**Comment by pi1541** [comment]
Added in v1.21

**Comment by svenpetersen1965** [comment]
I have tested the new firmware a bit. It works pretty good with the hardware buttons again (like v1.19 and unlike v1.20). The scrolling through the lists works kind of, but not perfectly: when I am at the element before the last and I scroll down, I don't get to the last, but the first ([..]). when I scorll up from the first, I get to the last, so I can circumvent this issue.   When I am in an directory with two elements, it is not possible to insert the last disk image with the insert button, but I can insert it by pressing the select button.

Other than that: pretty cool. I really like this new feature. 

I have also tested a couple (no, a whole bunch) of X'2018 demos, which all worked.
